config/core/test-configs.yaml

```diff
@@ -42,6 +47,12 @@ file_systems:
     nfs: 'qemu-{arch}/cip-core-image-kernelci-cip-core-buster-qemu-{arch}.tar.gz'
     root_type: nfs
 
+  chrome_version:
+    type: chrome-os
+    ramdisk: 'to be defined'
+    emmc: 'to be defined'
+    root_type: emmc
```

Comment:
We can't add the rootfs image for now, that will be done by the part of the job that does the flashing

Reply:
updated